app/matrix_bot/eventparser.py

```diff
@@ -75,6 +86,17 @@ def only_on_join(self) -> None:
         if not self.event.source.get("content", {}).get("membership") == "invite":
             raise EventNotConcerned
 
+    async def only_allowed_sender(self) -> None:
+        """
+        :raise EventNotConcerned: if the sender is not allowed to send messages
+        """
+        if not self.is_sender_allowed():
+            await self.matrix_client.send_markdown_message(
+                self.room.room_id,
+                "Albert n'est pas encore disponible pour votre domaine. Merci de rester en contact, il sera disponible après un bêta test !",
+            )
+            raise EventNotConcerned
+
```

Comment:
We could add a "Contact us contact@albert... is you need an access" ?

Reply:
No, we don't want access requests yet as we don't have enough resources to manage the requests yet.

Reply:
all right